### PR TITLE
Merge css for tags

### PR DESCRIPTION
**Pull request by stephywells**
Replace frm-cta-tag and frm-shift-key classes with frm-meta-tag
Remove frm-plan-required-tag since it's not used any more
Add: frm-red-tag, frm-grey-tag, frm-green-tag, frm-orange-tag

@shervElmi it looks like a lot of this was code from your PRs. Would you mind looking over this to make sure I didn't miss anything?

After:
Screenshot 2024-01-04 at 4 18 29 PM
Screenshot 2024-01-04 at 4 05 57 PM
Screenshot 2024-01-04 at 4 05 49 PM

### CODE REVIEW

File: css/frm_admin.css [hunk 1]
```diff
@@ -2797,27 +2797,38 @@ body.frm-body-with-open-modal {
...
+.frm-meta-tag.frm-grey-tag,
+.frm-meta-tag.frm-grey-tag:hover {
+	background-color: var(--sidebar-hover);
+	color: var(--grey-900);
+}
+
+.frm-meta-tag.frm-red-tag,
+.frm-meta-tag.frm-red-tag:hover {
 	background: #FDE6DE;
 	color: #F15A24;
```
Comment (shervElmi) [inline css/frm_admin.css:2819]: We should change the syntax to `color: var(--orange);` here.

File: css/frm_admin.css [hunk 2]
```diff
@@ -2797,27 +2797,38 @@ body.frm-body-with-open-modal {
...
 }
 
-.frm-meta-tag {
-	font-size: 10px;
```
Comment (shervElmi) [inline css/frm_admin.css:2814]: It's most likely an improvement here, but we have a minor change in typography. Here are the before and after:

### Before:
image

### After:
image

File: css/frm_admin.css (hunk 1 above)
Comment (stephywells) [reply css/frm_admin.css]: Good catch! This is fixed and uses the right colors for red.

**Review by garretlaxton — APPROVED** [review]

**Review by Crabcyborg — APPROVED** [review]
This looks good to me!

Thanks @stephywells @shervElmi and @garretlaxton for working on this!

🚀

**Pull request merged**